CHANGELOG.md

```diff
@@ -19,6 +27,7 @@ These should not affect the functionality of the module.
 
 **Merged pull requests:**
 
+- Release of 4.2.3 [\#269](https://github.com/voxpupuli/puppet-firewalld/pull/269) ([trevor-vaughan](https://github.com/trevor-vaughan))
```

Comment:
Could you label this one skip-changelog and remove this line?

Reply:
Sure